### DISCUSSION TITLE
Filters

### DISCUSSION THREAD

**Discussion by vovtornick** [python-telegram-bot/python-telegram-bot#4345] [Q&A]
Добрый день, работая с вашей библиотекой наткнулся на ошибку в коде, кидал код ИИ, говорит проблема в библиотеке python-telegram-bot, что его нужно обновить или что-то подобное, я обновлял, переустанавливал, проблема не пропала.
Ниже предоставлю сам код для ясности.
![image](https://github.com/python-telegram-bot/python-telegram-bot/assets/161504017/1eb35c03-b283-4773-927e-f1db30c58499)

**Comment by Poolitzer** [comment]
We only help in english, revoke your bot token and if you run a version of this library > 20 you need to code your project accordingly. AI is mostly trained on V13

**Reply by vovtornick** [reply]
Thanks.
